### PR TITLE
Add minimal h-entry support for indieweb

### PR DESCRIPTION
**Pull request by kevinmarks**
See http://indiewebcamp.com/h-entry for details and http://indiewebify.me for how to check this. More advanced support would be ISO-formatted dt-published, and including author URL and photo for richer attribution

### CODE REVIEW

**Comment by tarleb** [comment]
This seems to be very specific microformat. Couldn't the same result be reached using more general and W3C approved RDFa annotations?

**Comment by dissolve** [comment]
Its just minimal microformats semantic markup, doesn't work for just indieweb.

Don't want to get in to arguments of formats.  Theres a lot more than just RDFa and Microformats.

If there were a desire by users to have RDFa or some other format, that could easily add PRs for that.

**Comment by kevinmarks** [comment]
As discussed at the first link, h-entry is a general microformat for making up a page with author, date, title, summary and content and thus maps well to this generic template. The h-entry microformat has wide support across dozens of consuming implementations (see http://indiewebcamp.com/Webmention#IndieWeb_Examples ).

**Comment by tarleb** [comment]
Apologies for the long wait for another reply. I'm sympathetic to the goals of indieweb and think h-entry looks like a nice and quite useful format.  I'd still argue that it's too much of a niche format to warrant inclusion in a template which aims for generality. E.g. the [h-entry wiki site](https://indieweb.org/h-entry) states

> h-entry is the microformats2 vocabulary for marking up blog posts on web sites. It can also be used to mark-up any other episodic or time series based content. 

This doesn't necessary apply to all documents. Wrong markup is often worse than no markup at all.  To me, it's not even clear if something like [CreativeWork](http://1.schemaorgae.appspot.com/CreativeWork) from schema.org would be general enough to describe a document.

Regardless, semantic markup is an important topic and proper support for it would be very valuable IMHO.   It would probably require more changes to the HTML and HTML5 templates, additional documentation, and more thought on how to deal with ambiguities.  The `date` value, for example, can mean very different things, like publication date, creation date, modification date, or – as it could be free text – a combination of all of the above.

In summary I think it would be better for now to add a note in the [pandoc wiki](https://github.com/jgm/pandoc/wiki), explaining how metadata can be added to HTML output and what one would have to look out for.
